support/rmt

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#############################################################
+# Name:        Supportconfig Plugin for RMT
+# Description: Gathers important troubleshooting information
+#              about RMT
+# License:     GPLv2
+# Author:      SCC Team <happy-customer@suse.de>
+# Modified:    2018 April 18
+#############################################################
+
+SVER=0.0.1
+RCFILE="/usr/lib/supportconfig/resources/scplugin.rc"
+
+[ -s $RCFILE ] && . $RCFILE || { echo "ERROR: Initializing resource file: $RCFILE"; exit 1; }
+
+## Our own helper functions
+validate_rpm_if_installed() {
+  THISRPM=$1
+  echo "#==[ Validating RPM ]=================================#"
+  if rpm -q "$THISRPM" >/dev/null 2>&1; then
+    echo "# rpm -V $THISRPM"
+
+    if rpm -V "$THISRPM"; then
+      echo "Status: Passed"
+    else
+      echo "Status: WARNING"
+    fi
+  else
+    echo "package $THISRPM is not installed"
+    echo "Status: Skipped"
+  fi
+  echo
+}
+
+RPMLIST="
```

Comment:
Why not putting them in one line or even more modern using array notation to make clear this is an array and not a list. Wondering why this is working anyway since IFS should be set to whitespace.

Reply:
Noted